src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { forwardRef } from 'react';
+import cx from 'classnames';
+
+import Icon from 'Icon';
+import Label from 'Label';
+
+type CheckboxProps = {
+  checked?: boolean;
+  className?: string;
+  describedBy?: string;
+  disabled?: boolean;
+  id: string;
+  isInvalid?: boolean;
+  isRequired?: boolean;
+  label?: string;
+  name?: string;
+  onChange?: () => void;
+  value?: string;
+};
+
+export const Checkbox = forwardRef(
+  (
+    {
+      checked,
+      className,
+      describedBy,
+      disabled,
+      id,
+      isInvalid,
```

Comment:
As I mentioned in the PR description I built this component in the same way as `RadioInput`. I am not sure when we would like to use `isInvalid` prop. Is it the same as `disabled` or is is related to error messgae? Plus we don't have styles for `cc-checkbox--is-invalid` nor `cc-radio-input--is-invalid`. Should we keep it then?

Reply:
`isInvalid` would indicate to me that there is an error with this input - it should/should not be checked given certain conditions

Reply:
I realise I didn't answer the question - I'd say it definitely belongs with the component. Perhaps create another ticket to address invalid styles and prompt the designers for some guidelines or creative.